### ISSUE TITLE
Compatible with New Desktop for MATLAB (Beta) Add-On?

### ISSUE THREAD

**Issue by alexbastuba** [scottclowe/matlab-schemer#44]
Hello, 

I enjoy using your schemer to implement the dracula color scheme. However it seems this package is not compatible with the new desktop. I tried reenabling dracula (which functioned in vanilla matlab) but it doesn't seem to work anymore in the new dark mode desktop.

Does anyone else encounter this problem or is it something local on my end that is causing my theme to no longer load on the dark mode desktop? The matlab developers mentioned it uses a different engine based on Matlab Online if that offers any insight.

Thank you!

**Comment by toshiakit** [comment]
Yes, I am a Community Engagement Engineer at MathWorks and I would like to discuss this with you, @scottclowe, because there is a new API under consideration that can help make MATLAB Schemer compatible with the New Desktop.  Please reach out to me via my MATLAB Central profile https://www.mathworks.com/matlabcentral/profile/authors/951521/ (Click on "Message") button.

**Comment by auxym** [comment]
Hi @toshiakit! If you guys are developing a new GUI, as a suggestion, could you maybe support a built-in way importing color schemes from a standard json-like format? Almost all modern IDEs/editors support this.

**Comment by HaHeho** [comment]
Left another comment regarding this in the plugin discussion section at  https://www.mathworks.com/matlabcentral/fileexchange/53862-matlab-schemer. Hoping for an update.

**Comment by auxym** [comment]
@HaHeho FYI, Mathworks has released this for R2025+: https://www.mathworks.com/matlabcentral/fileexchange/181402-matlab-color-theme-extensions.

And I made my own, too: https://github.com/tinted-theming/tinted-matlab
